pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1864,10 +1864,12 @@ class Timedelta(_Timedelta):
 
     Parameters
     ----------
-    value : Timedelta, timedelta, np.timedelta64, str, or int
+    value : Timedelta, timedelta, np.timedelta64, str, int or float
         Input value.
     unit : str, default 'ns'
-        Denote the unit of the input, if input is an integer.
+        If input is an integer, denote the unit of the input.
+        If input is a float, denote the unit of the integer parts.
+        (The decimal parts with an accuracy higher than 1 nanosecond are ignored.)
```

Comment:
I think "accuracy" would be better termed "resolution", and in which case, it's decimal parts with a resolution lower than 1 nanosecond are ignored. I also think the parentheses can be removed.

Reply:
Thanks a lot for your comment. I'm sorry I reinstalled the system so I lost the pull request. I opened a new pull request to change it. Thanks again !!!

Reply:
This is fine for now, but in the future you did not lose the branch nor pull request. It is still available:

https://github.com/ZKaoChi/pandas/branches

You can get it by `git switch origin Solution-for-issue-#60044`

Reply:
I've changed the wording under this pull request. Could you please review it again?